src/cosmoz-dropdown.ts

```diff
@@ -46,6 +46,19 @@ const Dropdown = (host: HTMLElement & DropdownProps) => {
 	const { placement, render } = host,
 		anchor = useCallback(() => host.shadowRoot!.querySelector('.anchor'), []),
 		{ active, onToggle } = useHostFocus(host);
+
+	useEffect(() => {
+		const toggleActionList = async () => {
+			if (host.getAttribute('id') === 'cosmoz-bottom-bar-action-menu-btn') {
+				host.shadowRoot?.getElementById('dropdownButton')?.click();
+			}
```

Comment:
Using the ID in this fashion is very unintuitive. Altering the behavior of a component based on the ID that it was assigned is very problematic and should be avoided. The ID is meant for identifying a particular element in the context of a page/view/app/component that uses that element. Do not get creative with it and stick to it's purpose. 

If you want to alter behaviour, you should add an attribute/property that controls that behaviour. The name of the property should reflect the purpose. In this case it could be `openOnToggleActionList`, and you would use it like so:
```js
html`<cosmoz-dropdown .openOnToggleActionList=${true} [...]>`
```

The fact that it's hard to give a name that doesn't sound weird is a hint that this is not the most elegant solution.